app/screens/post_options/post_options.tsx

```diff
@@ -127,7 +127,7 @@ const PostOptions = ({
             closeButtonId={POST_OPTIONS_BUTTON}
             componentId={Screens.POST_OPTIONS}
             initialSnapIndex={0}
-            snapPoints={[((snapPoints + additionalSnapPoints) * ITEM_HEIGHT), 10]}
+            snapPoints={[((snapPoints + additionalSnapPoints) * MENU_ITEM_HEIGHT), 10]}
```

Comment:
You are using MENU_ITEM_HEIGHT (50) from constants, but OptionItem defines also an ITEM_HEIGHT (48).

Shouldn't we use the one from OptionItem? (and make sure the one in OptionItem is properly calculated, and not so much a magic number)

When I say having it calculated, I mean stuff like creating all the necessary intermediate constants to be used in the styles that sum up to the actual height of the item.

Reply:
The OptionItem component adheres to the figma design... the  MENU_ITEM_HEIGHT (50) is not correct.  I'm removing it in favor the the ITEM_HEIGHT.